### DISCUSSION TITLE
Add multiple boxplots for single row

### DISCUSSION THREAD

**Discussion by Julie-Cardin** [DingWB/PyComplexHeatmap#77] [Q&A]
Hi,
  I wonder if it is possible to have more than 1 box in box plot annotation per column of heatmap? Like 1 color for treament and 1 color for control for a same sample placed together side by side. One pair of boxes per sample. Please.
  
Many thanks, 
Ex: looking to plot something like this:

![Rplot01_hmp_Boxplot_2categories](https://github.com/DingWB/PyComplexHeatmap/assets/118992213/3a5c6e6f-67f6-4318-a5b0-0390be6c9613)

**Comment by DingWB** [comment]
How about the anno_lineplot and anno_barplot (stacked barplot) in [this example](https://dingwb.github.io/PyComplexHeatmap/build/html/notebooks/advanced_usage.html#Only-plot-the-annotations).
In general, the width of columns is very small in a heatmap, so it would be difficult to show multiple columns of boxplot in one heatmap column. 
So, I don't think it is worthy to implement this feature. But users can always customize it themselves. For example, 
the following code could be added to the end of [this example](https://dingwb.github.io/PyComplexHeatmap/build/html/notebooks/advanced_usage.html#Add-selected-rows-labels) (before plt.savefig) to clear plotted figure and then plot whatever you want using `ax`
```python
for ax in cm.top_annotation.axes[-1,:]:
    ax.cla()
    # pot whatever you want using ax
```

**Comment by Julie-Cardin** [comment]
Hi, I didn't meant to ask for another feature implementation, sorry if it looks like that. I thought maybe it is possible to do what I want with actual methods, but I searched and I didn't found. So my question for you was more to have a hint if it is possible? And if yes, if you had a suggestion what to use.
My Goal is to be able to show  2 boxes side-by-side for 2 categories/treatments  per sample, with the Logo associated.
I use PyCompexHeatmap because was great to be able to assemble Logos 

So should have been more specific with my example. Sorry.

![Rplot01_hmp_Boxplot_2categories](https://github.com/DingWB/PyComplexHeatmap/assets/118992213/423b0182-be3a-4471-ad58-ca8ff10c217d)
